apps/mark/frontend/README.md

```diff
@@ -1,26 +1,14 @@
 # VotingWorks Ballot Marking Device (BMD)
 
-Marks and/or prints ballots for a voter.
+Marks and prints ballots for a voter.
 
 ## Setup
 
 Follow the instructions in the [VxSuite README](../../README.md) to get set up,
 then run BMD like so:
 
 ```sh
-# In frontends/bmd
-# To run in mark-only mode (when there's a separate standalone printer)
-VX_APP_MODE=MarkOnly pnpm start
-
-# To run in print-only mode (for a standalone printer)
-VX_APP_MODE=PrintOnly pnpm start
-
-# To run in mark-and-print mode (for a BMD with its own printer attached)
-VX_APP_MODE=MarkAndPrint pnpm start
-
-# By default, the BMD runs in mark-and-print mode
```

Comment:
@carolinemodic does this mean we should remove the relevant content from `vxsuite-complete-system`?